ts/features/itwallet/common/components/discoveryBanner/ItwDiscoveryBanner.tsx

```diff
@@ -62,18 +73,44 @@ export const ItwDiscoveryBanner = ({
     dispatch(itwCloseDiscoveryBanner());
   };
 
+  // This effect is used to determine which banner to show and if it should be shown
+  useEffect(() => {
+    const isDataLoaded = pipe(O.fromNullable(isWalletRemotelyActive), O.isSome);
+    if (isDataLoaded) {
```

Comment:
This can be simplified. What do you think?
```suggestion
    if (isWalletRemotelyActive) {
```